### DISCUSSION TITLE
油管 WPF 领域博主推荐

### DISCUSSION THREAD

**Discussion by BYJRK** [BYJRK/DotNet-Discussions#3] [WPF 开发]
> 注意，这里推荐的是 WPF 领域的，其他还有一些 .NET 领域的，这里暂不列出。

## AngelSix

https://www.youtube.com/@AngelSix

有一个非常棒的 WPF 开发系列视频（MVVM）：[WPF UI Programming](https://www.youtube.com/playlist?list=PLrW43fNmjaQVYF4zgsD0oL9Iv6u23PI6M)

最近他在做 Avalonia UI 的开发：[Avalonia UI](https://www.youtube.com/playlist?list=PLrW43fNmjaQWMhFHxS1jpQ34TkHroHJLb)

## SingletonSean

https://www.youtube.com/@SingletonSean

做过很多 WPF 开发、MVVM 相关的视频，包括对于概念与知识的科普等，但基本不用 MVVM 工具包，完全自己实现

## Brian Lagunas

https://www.youtube.com/@BrianLagunas

知名 MVVM 工具包 Prism 的作者，微软 MVP，做过一些 WPF 开发的直播实况，以及 WPF 问题答疑

## Tim Corey

https://www.youtube.com/@IAmTimCorey

视频涉及内容非常丰富，有 WPF、C# 知识与技巧、.NET 新特性等等

## James Montemagno

https://www.youtube.com/@JamesMontemagno

虽然做的主要是 MAUI 开发（以前主要是 Xamarin），但是很多内容（尤其是 MVVM 相关）与 WPF 是相通的

## C# Design Pro

https://www.youtube.com/@CSharpDesignPro

视频内容主要为如何用原生 WPF 制作出各种华丽的效果

## 其他待补充

**Comment by N4v1zzZ** [comment]
### Kevin Bost
https://www.youtube.com/@Kitokeboo

Material Design的作者，经常会分享一些开发MD的日常和一些WPF相关的东西。
